common/views/components/DefaultPageLayout/DefaultPageLayout.js

```diff
@@ -15,13 +18,38 @@ export type JsonLdObject = {
   "@type": string
 }
 type jsonData = {
-  pageJsonLd: JsonLdObject
+  data: JsonLdObject
+};
+
+const galleryOpeningTimes = function(galleryHours: ?OpeningHours) {
+  if (galleryHours) {
+    return {
+      openingHoursSpecification: galleryHours && galleryHours.regular.map(
+        openingHoursDay =>  {
+          const specObject = objToJsonLd(openingHoursDay, 'OpeningHoursSpecification', false);
+          delete specObject.note;
+          return specObject;
+        }
+      ),
+      specialOpeningHoursSpecification: galleryHours.exceptional && galleryHours.exceptional.map(
+        openingHoursDate => {
+          const specObject = {
+            opens: openingHoursDate.opens,
+            closes: openingHoursDate.closes,
+            validFrom: moment(openingHoursDate.overrideDate).format('DD MMMM YYYY'),
```

Comment:
Should be in [ISO 8601 date format](https://en.wikipedia.org/wiki/ISO_8601). From https://schema.org/Date.